vamp-plugins/dsp/TCSgram.cpp

```diff
@@ -23,8 +23,13 @@
 #include "MathUtilities.h"
 
 TCSGram::TCSGram() :
-	m_uNumBins(6)
+	m_uNumBins(6),
```

Comment:
For 3rd-party code I'd prefer we not do style changes -- that way when we're trying to merge upstream changes we know all of the lines we changed in the files were code changes and not style/indentation.